rllib/policy/sample_batch.py

```diff
@@ -804,7 +804,7 @@ def to_device(self, device, framework="torch"):
     def size_bytes(self) -> int:
         """Returns sum over number of bytes of all data buffers.
 
-        For numpy arrays, we use `.nbytes`. For all other value types, we use
+        For numpy arrays, we use ``.nbytes``. For all other value types, we use
```

Comment:
This accidentally links to .nbytes in datasets

Reply:
resolved offline